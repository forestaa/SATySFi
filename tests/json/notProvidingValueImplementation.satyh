module X : sig
  val x : int
end = struct
end
