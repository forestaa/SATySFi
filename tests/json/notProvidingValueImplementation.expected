{
  "file": "notProvidingValueImplementation.satyh",
  "range": {
    "start": { "line": 1, "character": 0 },
    "end": { "line": 4, "character": 3 }
  },
  "message":
    "! [Type Error] The implementation does not provide value 'x',\n    which is required by the signature."
}
